MdePkg/Include/IndustryStandard/IoRemappingTable.h

```diff
@@ -59,10 +59,13 @@
 #define EFI_ACPI_IORT_SMMUv1v2_INT_FLAG_LEVEL  0x0
 #define EFI_ACPI_IORT_SMMUv1v2_INT_FLAG_EDGE   0x1
 
-#define EFI_ACPI_IORT_SMMUv3_FLAG_COHAC_OVERRIDE    BIT0
-#define EFI_ACPI_IORT_SMMUv3_FLAG_HTTU_OVERRIDE     BIT1
-#define EFI_ACPI_IORT_SMMUv3_FLAG_PROXIMITY_DOMAIN  BIT3
-#define EFI_ACPI_IORT_SMMUv3_FLAG_DEVICEID_VALID    BIT4
+// MU_CHANGE Start
+#define EFI_ACPI_IORT_SMMUv3_FLAG_COHAC_OVERRIDE        BIT0
+#define EFI_ACPI_IORT_SMMUv3_FLAG_HTTU_OVERRIDE_AF      BIT1  // HW update of Access Flag supported

```

Comment:
Renaming this define might lead to breaking change (depending on whether platforms are consuming this). Do we really need to make the "_AF" into the name?

Reply:
@kuqin12 we could instead do:

```
#define EFI_ACPI_IORT_SMMUv3_FLAG_HTTU_OVERRIDE         BIT1  // HW update of Access Flag supported
#define EFI_ACPI_IORT_SMMUv3_FLAG_HTTU_OVERRIDE_DS   BIT2  // HW update of Access Flag + Dirty Flag supported
```

So the "Access Flag" portion of the override is implied. The comment next to the define should be clear enough

Reply:
Sure, that would work. Thanks!